dev/tools/localization/gen_l10n.dart

```diff
@@ -241,7 +241,12 @@ String genPluralMethod(Map<String, dynamic> bundle, String key) {
   for(String pluralKey in pluralIds.keys) {
     final RegExp expRE = RegExp('($pluralKey){([^}]+)}');
     final RegExpMatch match = expRE.firstMatch(message);
-    if (match.groupCount == 2) {
+    if (match == null) {
```

Comment:
This works assuming that the `other` argument is required, like in English. 

Ideally, we'd want to be able to support any combination of numeric values or rules, since the [ICU message syntax](http://userguide.icu-project.org/formatparse/messages) doesn't seem to explicitly define which exact ones to use (probably intentionally, since different languages would have different plural rules and matches).

This should be okay for now. Could you add a TODO where it makes sense in this method that we should write more flexible logic that accounts for any combination of plural matchers?

Reply:
Makes sense to not have the `other` argument required.

I'm just adding the null check now, so that the tool continues if we miss one of the cases.

Reply:
@perclasson and I spoke outside of the PR and we could just have the tool skip any arguments that do not have a matcher. This gives the developers the freedom to decide what plural cases they need.